dashboard/app/helpers/levels_helper.rb

```diff
@@ -237,16 +237,21 @@ def blockly_options(local_assigns={})
       open_function_definition
       callout_json
       disable_sharing
+      hide_source
+      share
+      no_padding
+      show_finish
+      embed
     ).map{ |x| x.include?(':') ? x.split(':') : [x,x.camelize(:lower)]}]
     .each do |dashboard, blockly|
       # Select first valid value from 1. local_assigns, 2. property of @level object, 3. named instance variable, 4. properties json
       # Don't override existing valid (non-nil/empty) values
-      property = local_assigns[dashboard].presence ||
+      property = local_assigns[dashboard.to_sym].presence ||
```

Comment:
I don't think this worked correctly before.  Symbolizing the string makes local_assigns lookups to work as described in the comment.